packages/dart/noports_core/lib/src/sshnp/util/srvd_channel/srvd_channel.dart

```diff
@@ -135,6 +135,10 @@ abstract class SrvdChannel<T> with AsyncInitialization, AtClientBindings {
     srvdAck = SrvdAck.notAcknowledged;
     subscribe(regex: '$sessionId.${Srvd.namespace}@', shouldDecrypt: true)
         .listen((notification) async {
+      if (fetched) {
+        logger.warning('Got additional relay response ${notification.value} - ignoring');
+        return;
+      }
```

Comment:
fix the client-side race condition; not strictly necessary now that the feature exists in the relay, but should absolutely be here anyway